### PR TITLE
Add account username in reset password email

### PR DESCRIPTION
**Pull request by marcocaberletti**
This PR resolve issue #108.
Main changes:
- edit mail template;
- add username field into model;
- add unit test.

### CODE REVIEW

File: iam-login-service/src/main/resources/templates/resetPassword.vm [hunk 1]
```diff
@@ -4,5 +4,6 @@ Follow this link to set a new password for your $organisationName account:
 
 $resetPasswordUrl
 
+After updating the password, you can login into IAM with the username: $username
```
Comment (andreaceccanti) [inline iam-login-service/src/main/resources/templates/resetPassword.vm]:
```
Once your password has been reset, you can login to the IAM using the following username:

$username
```

File: iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java [hunk 2]
```diff
@@ -107,17 +115,17 @@ public void testSendEmails() throws MessagingException {
     RegistrationRequestDto reg = createRegistrationRequest(username);
     notificationService.sendPendingNotifications();
 
-    Assert.assertEquals(1, wiser.getMessages().size());
+    assertEquals(1, wiser.getMessages().size());
```
Comment (andreaceccanti) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java]: Use assertThat and equalTo
```java
assertThat(wiser.getMessages(), hasSize(1))
```

Comment (andreaceccanti) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java]: same holds for other checks, always try to use assertThat and hamcrest matchers that provide clearer error messages

File: iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java [hunk 3]
```diff
@@ -367,17 +371,40 @@ public void testAdminNotificationMailShouldContainsDashboardLink()
...
-    Assert.assertTrue("text/plain", message.getMimeMessage().isMimeType("text/plain"));
+    assertTrue("text/plain", message.getMimeMessage().isMimeType("text/plain"));
+    String content = message.getMimeMessage().getContent().toString();
+    assertThat(content, containsString(dashboardUrl));
+
+    deleteUser(reg.getAccountId());
+  }
+
+  @Test
+  public void testPasswordResetMailShouldContainsUsername() throws MessagingException, IOException {
```
Comment (andreaceccanti) [inline iam-login-service/src/test/java/it/infn/mw/iam/test/notification/NotificationTests.java]: rename test to testPasswordResetMailContainsUsername

**Comment by marcocaberletti** [comment]
Test me please

**Comment by marcocaberletti** [comment]
test this please

**Pull request merged**